modules/swift/examples/code_snippets/SampleCodeTest.swift

```diff
@@ -779,6 +779,23 @@ class SampleCodeTest {
         // end::replication-pull-filter[]
     }
 
+//  Added 2/Feb/21 - Ian Bridge
+    func testCustomHeartbeat() {
+        // tag::replication-set-heartbeat[]
+        let target =
+          URLEndpoint(url: URL(string: "ws://foo.couchbase.com/db")!)
+
+        let config =
+          self.config(target: target,
+            type: .pushAndPull,
+            continuous: true)
+
```

Comment:
same like above, can we use simple config initialization?

Reply:
@jayahariv see https://github.com/couchbase/docs-couchbase-lite/pull/401/commits/109d058b0d5750d2cb325ae9bbccc5edffd527fe